pkg/apis/agones/v1/common.go

```diff
@@ -175,14 +175,15 @@ func (ao *AllocationOverflow) Apply(gs *GameServer) {
 	}
 }
 
-// Priority is a sorting option for GameServers with Counters or Lists based on the Capacity.
+// Priority is a sorting option for GameServers with Counters or Lists based on the available capacity,
```

Comment:
We use priorities in two different places in two different ways. The fleet autoscaler uses total capacity  (see comment in fleet.yaml), and the game server allocation uses available capacity.

https://github.com/googleforgames/agones/blob/589f77b4bddac84b6ab225cb62b44aa181b53cbd/pkg/gameserverallocations/allocation_cache.go#L297-L302